public/js/authservice.js

```diff
@@ -0,0 +1,102 @@
+// app.js
+
+window.addEventListener('load', function() {
+  let idToken;
+  let accessToken;
+  let expiresAt;
+  if (!location.href.split('#')[0].endsWith('/redirect')){
+    localStorage.destURL = location.href;
+  }
+  var webAuth = new auth0.WebAuth({
+    domain: 'participedia.auth0.com',
+    clientID: 'lORPmEONgX2K71SX7fk35X5PNZOCaSfU',
+    responseType: 'token id_token',
+    scope: 'openid profile email picture user_metadata',
+    redirectUri: location.origin + "/redirect"
+  });
+
+  var loginBtns = document.querySelectorAll('.loginButton');
```

Comment:
i've been using the pattern of using class names prepended with `js-` to distinguish classes used for style and classes used for behaviour, so i would suggest changing this to to `js-login-button` for consistency.